recipes/scine-utilities-python/meta.yaml

```diff
@@ -0,0 +1,96 @@
+{% set name = "utilities" %}
+{% set pkg_name = "scine-" + name + "-python" %}
+{% set version = "4.0.0" %}
+
+package:
+  name: {{ pkg_name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/qcscine/utilities/archive/refs/tags/{{ version }}.tar.gz
+    sha256: 3bcabd8f4270f0c1efd235a0d22506f4750b0ccffa99184d4362b95a1da7a1d4
+    fn: {{ name }}.tar.gz
+    patches:
+      - skip_library_build.patch
+  - url: https://github.com/qcscine/development-utils/archive/refs/tags/4.0.0.tar.gz
+    sha256: 54002c2082b6bb75672ec66bf9cf3935bbdf6b085ed9b4d7174cbdedb7c2275d
+    fn: dev.tar.gz
+    folder: dev
+    patches:
+      - pybind11_pin.patch
+
+build:
+  number: 0
+  skip: true  # [win or osx]
+  script:
+    - set -ex
+    - >
+       cmake
+       ${CMAKE_ARGS}
+       -B _build -G Ninja
+       -DPYTHON_EXECUTABLE={{ PYTHON }}
+       -DSCINE_MARCH=""
+       -DSCINE_SKIP_LIBRARY=ON
+       -DSCINE_BUILD_TESTS=OFF
+       -DSCINE_BUILD_PYTHON_BINDINGS=ON
+    - cmake --build _build
+    - cmake --install _build
+
+  run_exports:
+    - {{ pin_subpackage(pkg_name, max_pin='x') }}
```

Comment:
This package contains the python bindings only for scine? If so, it should not have a run_export because it should never be compiled against only imported as a python module.
```suggestion
```